Embedded/MaxMix/Structs.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "Config.h"
+
+typedef struct __attribute__((__packed__))
+{
+  char name[ITEM_BUFFER_NAME_SIZE]; // 36 Bytes (36 Chars)
+  uint32_t id;                      // 4 Bytes (32 bit)
+  int8_t volume;                    // 1 Byte
```

Comment:
should vol be unsigned?

Reply:
doesn't matter, volume range is 0-100 capped, which is in the range for both signed and unsigned.

Reply:
Yea ok. I just thought to make it consistent cuz the other functions passes vol as unsigned.

Reply:
Ya, I am afraid of changing it to uint8_t as depending on all the math working on volume, it could introduce a new issue. Will need to review all usage of volume in the firmware before changing it.